Source/Login/Backend/identities/tenants/getter.go

```diff
@@ -4,15 +4,25 @@ type Getter interface {
 	GetTenantFromID(tenantID TenantID) (*Tenant, error)
 }
 
-func NewGetter() Getter {
-	return &getter{}
+func NewGetter(configuration Configuration) Getter {
+	return &getter{
+		configuration,
+	}
 }
 
-type getter struct{}
+type getter struct {
+	configuration Configuration
+}
 
 func (g *getter) GetTenantFromID(tenantID TenantID) (*Tenant, error) {
```

Comment:
Changing the name, and suggestion (can't put direct suggestion here sorry)

```go
func (g *getter) GetTenantFromID(tenantID TenantID) (*Tenant, error) {
	tenant := &Tenant{
		ID: tenantID,
		Display: string(tenantID),
	}

	if name, ok := g.configuration.TenantNames()[tenantID]; ok {
		tenant.Display = name
	}

	return tenant, nil
}
```

Reply:
Fixed in [this commit](https://github.com/dolittle-platform/Authentication/pull/8/commits/7a1fde4587086b88dc5bd09e77973e1a4bb01819)